about.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
+    <title>About</title>
+</head>
+<body id="body">
+    <header class="header">
+        <ul class="mobile-hidden social-media-top">
+            <li class="fa fa-facebook fb" ></li>
+          <li class="fa fa-twitter twitter"> </li>
+            <li><p class="social-media-top-item">English</p> </li>
+           <li> <p class="social-media-top-item">My page</p></li>
+           <li> <p class="social-media-top-item">Logout</p></li>
+                 </ul>
+                 <nav class="navigation">
+        <div id="cover">
+            <img id="close" class="dektop-hidden" src="./assets/images/close.png" alt="close window">
+            <ul id="menu" class="dektop-hidden">
+                <li ><a  class="menu-item" href="./index.html">Home</a></li>
+                <li ><a  class="menu-item" href="#">Our Blog</a></li>
+                <li ><a  class="menu-item" href="#">Join</a></li>
+                <li ><a  class="menu-item" href="#">Sponsor</a></li>
+            </ul>
```

Comment:
- [ ] When I open the mobile menu and switch to the desktop screen the mobile menu is still on the screen it should disappear. Please make sure when we switch the screen from mobile to desktop the mobile menu should disappear.
 - You can use media queries for that if the screen cross the min-width you can set your mobile menu `display: none`

See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157938553-491d3f46-4be0-4d28-9b88-209a6573cee5.png)